webapp/src/components/properties/lastModifiedAt/lastModifiedAt.tsx

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react'
+
+import {Card} from '../../../blocks/card'
+import {CardTree} from '../../../viewModel/cardTree'
+
+const moment = require('moment')
+
+type Props = {
+    card: Card,
+    cardTree: CardTree | undefined
+}
+
+const LastModifiedAt = (props: Props): JSX.Element => {
+    let latestBlock = props.cardTree ? props.cardTree.allBlocks.filter((block) => block.parentId === props.card.id || block.id === props.card.id).sort((a, b) => b.updateAt - a.updateAt)[0] : props.cardTree
+    latestBlock = latestBlock || props.card
+
+    return (
+        <div className='LastModifiedAt octo-propertyvalue'>{moment(latestBlock.updateAt).format('llll')}</div>
```

Comment:
nit:

can we multiline this for readability?
```suggestion
        <div className='LastModifiedAt octo-propertyvalue'>
              {moment(latestBlock.updateAt).format('llll')}
       </div>
```

Reply:
Done.